gravitee-apim-portal-webui-next/src/entities/subscription/subscription.fixture.ts

```diff
@@ -50,6 +50,11 @@ export function fakeSubscriptionResponse(
     links: {
       self: 'test',
     },
+    metadata: {
```

Comment:
Since the application id should always exist with the name, it should be added here to reflect in the tests.

Reply:
Done